src/higley_lab_to_nwb/benisty_2022/benisty_2022_spike2ttlinterface.py

```diff
@@ -0,0 +1,93 @@
+from typing import List
+import numpy as np
+from neo import io
+
+from neuroconv.datainterfaces.ecephys.baserecordingextractorinterface import BaseRecordingExtractorInterface
+from neuroconv.tools import get_package
+from neuroconv.utils import FilePathType
+from neuroconv.tools.signal_processing import get_rising_frames_from_ttl
+from spikeinterface.extractors import CedRecordingExtractor
+def _test_sonpy_installation() -> None:
+    get_package(
+        package_name="sonpy",
+        excluded_python_versions=["3.10", "3.11"],
+        excluded_platforms_and_python_versions=dict(darwin=dict(arm=["3.8", "3.9", "3.10", "3.11"])),
+    )
+def get_stream_ids_and_names(file_path: FilePathType) -> List[str]:
```

Comment:
There is a precedence for this function and is called `get_streams` elsewhere so I'd keep it that way:
```suggestion
def get_streams(file_path: FilePathType) -> List[str]:
```